docs/guides/_toc.json

```
@@ -505,7 +505,25 @@
         },
         {
           "title": "Qiskit addons",
-          "url": "/guides/addons"
+          "children": [
+            {
+              "title": "Qiskit addons overview",
+              "url": "/guides/addons"
+            },
+            {
+              "title": "Sample-based Quantum Diagonalization (SQD)" ,
```

```suggestion
              "title": "Sample-based quantum diagonalization (SQD)" ,
```
Sentence case (but if someone wants to argue that it should stay capped before the acronym, I'm fine to leave it)